### DISCUSSION TITLE
0.28.0

### DISCUSSION THREAD

**Discussion by meikpiep** [meikpiep/gauguin#192] [Announcements]
# Added

- Translation into Traditional Chinese. (Huge thanks to @2013ZRZ!)

# Fixed

- Fix missing grid preview in new game screen, which occurred for some popular screen types.  (Reported by @gittesserakt)
- Fix generating rectangular grids.
- Fix equalizing best time to no longer be handled as a new best time. (Reported by @GhostShadow0316)

<hr /><em>This discussion was created from the release <a href='https://github.com/meikpiep/gauguin/releases/tag/v0.28.0'>0.28.0</a>.</em>

**Comment by zina0316** [comment]
I have to say, the translation of **Traditional Chinese** is terrible.
The translator isn't even a Taiwanese or Malaysian
He's Chinese, so his words are not what Taiwanese would use
He should only translate **Simplified Chinese** and not touch **Traditional Chinese**

**Reply by meikpiep** [reply]
I can't judge this as I do not speak but English and German.

@GhostShadow0316 @2013ZRZ and others with knowledge of Traditional Chinese: I really appreciate you to get in touch and collaborate, no matter if here or via Weblate. If there is anything I can do, let me know. Many thanks!

**Reply by 2013ZRZ** [reply]
@GhostShadow0316  You're right. I just converted Simplified Chinese characters into Traditional Chinese characters, so there may be some inaccuracies or inapplicability. If you can, please participate and help with localization. Thank you!
